assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -167,7 +167,13 @@ function MediaPane(props) {
    * @return {null|*} Return onInsert or null.
    */
   const insertMediaElement = useCallback(
-    (resource) => insertElement(resource.type, { resource }),
+    (resource, thumbnailURL) => {
+      window.webStoriesEditorResourceList[resource.id] = {
```

Comment:
Could we create some simple module providing this - just as an internal list? Or is it necessary to have accessible outside react?

Reply:
It's not needed, moved in https://github.com/google/web-stories-wp/pull/3935/commits/8edf8b089b4383a4753287ec4c0c670c3ed500d8